arch/arm/Kconfig

```diff
@@ -6,9 +6,16 @@
 menu "ARM Options"
 	depends on ARM
 
-source "arch/arm/core/Kconfig"
+source "arch/arm/core/aarch32/Kconfig"
+source "arch/arm/core/aarch64/Kconfig"
 
 config ARCH
 	default "arm"
 
+config ARM64
+	bool
+	prompt "ARM64 (AArch64)"
```

Comment:
This should not have a prompt unless necessary.

Presumably some option could select this instead of having this be a user-configuration option.

Reply:
ACK. Fix coming in V3. Closing to declutter the story. Thanks.